css/global.css

```diff
@@ -3,3 +3,7 @@ div.feed-shared-update-v2:has(div):has(div):has(a.app-aware-link):has(
   ) {
   display: none !important;
 }
+
+button[class*='feed-shared-coach-prompt'] {
```

Comment:
According to [this W3 Schools article](https://www.w3schools.com/css/css_attribute_selectors.asp), the asterisk after the attribute (in this case, the class name) means that the selected elements should have a class name that includes the designated value. 

In other words, this selects all button elements with a class name that includes `feed-shared-coach-prompts`. This was helpful because there were multiple classes that all contained these words.